Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -339,6 +340,7 @@ local questsToBlacklistBySoDPhase = {
         [90184] = true, -- Hiding Priest Homunculi Teldrassil for now as there are too many icons
         [90185] = true, -- Hiding Priest Homunculi Durotar for now as there are too many icons
         [90186] = true, -- Hiding Priest Homunculi Tirisfal Glades for now as there are too many icons
+        [90187] = true, -- Hiding Priest Shadow Word: Death Westfall for now as there are too many icons
```

Comment:
Instead of hiding this, we should cleanup the spawns of [Leprithus](https://www.wowhead.com/classic/npc=572/leprithus). 

Current spawns are:

```lua
{[40]={{43.04,27.0},{41.93,29.24},{42.15,30.13},{64.71,66.7},{64.93,67.92}}}
```

which should be cleaned up to be only two:

```lua
{[40]={{41.93,29.24},{64.71,66.7}}}
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5562/commits/ffade6e89c89378a6c9765f626b7a05e8267374e